analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/parser/ParserService.kt

```diff
@@ -0,0 +1,57 @@
+package de.maibornwolff.codecharta.tools.ccsh.parser
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptList
+import de.maibornwolff.codecharta.filter.edgefilter.EdgeFilter
+import picocli.CommandLine
+
+class ParserService {
+    companion object {
+        fun selectParser(commandLine: CommandLine): String {
+            val selectedParser: String = KInquirer.promptList(
+                message = "Which parser do you want to execute?",
+                choices = getParserNamesWithDescription(commandLine)
+            )
+
+            return extractParserName(selectedParser)
+        }
+
+        fun executeSelectedParser(selectedParser: String) {
+            val args = emptyArray<String>()
+
+            when (selectedParser) {
+                "check" -> print("Not implemented yet")
```

Comment:
Maybe we should add a new private helper function in this ParserService that just prints the "Not implemented yet" message => private fun printNotSupported(parserName: string) { ... }

And I would prefer a message like "The interactive usage of parser <Foo> is not supported yet. Please specify the full command to run the parser."

Reply:
I like this approach 👍  I implemented it in this way and slightly adjusted the tests.